tests/test_evaluation.py

```diff
@@ -4,16 +4,12 @@
     import common as c
 
 
-class TestSplinepyEvaluation(c.unittest.TestCase):
+class TestSplinepyEvaluation(c.SplineBasedTestCase):
     def setUp(self):
-        self.b2P2D = c.b2P2D.copy()
-        self.n2P2D = c.n2P2D.copy()
-        self.z2P2D = c.z2P2D.copy()
-        self.r2P2D = c.r2P2D.copy()
-        self.bspline = c.splinepy.BSpline(**self.b2P2D)
-        self.nurbs = c.splinepy.NURBS(**self.n2P2D)
-        self.bezier = c.splinepy.Bezier(**self.z2P2D)
-        self.rational = c.splinepy.RationalBezier(**self.r2P2D)
+        self.bspline = self.spline_b2p2d()
+        self.nurbs = self.spline_n2p2d()
+        self.rational = self.spline_r2p2d()
+        self.bezier = self.spline_z2p2d()
```

Comment:
This puts us in the same position as before

Reply:
If a spline is needed multiple times in each test case, I think it makes more sense to get them from the `SplineBasedTestCase` class at the beginning of the test case and use it throughout it.